### ISSUE TITLE
how to add images?

### ISSUE THREAD

**Issue by lb2410** [lb2410/lorenebouboushian.org#7]
yup, somehow can't figure this out even with markdown tutorials on the internet...

see where it says 'Undoing and Doing Notes' on this page?
https://lorenebouboushian.org/projects/UNDOINGANDDOING

well that should be alt text for a photo in the assets folder

also gotta center photos

**Comment by edwardsharp** [comment]
i fixed the image, it should be like: `![Undoing and Doing Notes](/assets/images/udnotesfromtaty.JPG)` 

there's a few examples how to do this in the README file (you can see it here: https://github.com/lb2410/lorenebouboushian.org)
